### PR TITLE
test: fix monmap name check

### PR DESCRIPTION
**Pull request by leseb**
Closes: #421, #419  and #420

Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
@alfredodeza

**Comment by leseb** [comment]
test this please

File: tests/base/test_monitors.py [hunk 1]
```diff
@@ -30,6 +30,12 @@ def test_keyring_exists(self, mon_containers, client):
         if client.exec_inspect(command)['ExitCode'] != 0:
             raise AssertionError(result)
 
+    def test_monmap_exists(self, mon_containers, client):
+        command = client.exec_create(mon_containers, cmd='stat /etc/ceph/monmap-ceph')
```
Comment (alfredodeza) [inline tests/base/test_monitors.py]: the reason I had this different for hammer/jewel is that some entrypoint scripts use `monmap-ceph` while others use `ceph.monmap`. Does this change mean that you expect to normalize all monmap files to follow this convention of `monmap-ceph` ?

Comment (leseb) [inline tests/base/test_monitors.py]: Correct. Now I see where the `ceph.monmap` comes from. It's only on the demo images. I'm going to normalize that.

**Comment by alfredodeza** [comment]
We don't have CI hooked up to call these tests. Merging as it looks OK to me

**Comment by leseb** [comment]
Thanks!

**Pull request merged**